src/index.ts

```diff
@@ -5,13 +5,15 @@ interface FormField<T> {
   error: string | false;
   disabled: boolean;
   dirty: boolean;
+  isEnabledValidation: boolean;
```

Comment:
nit: this sounds like more correct English grammar to me
```suggestion
  validationEnabled: boolean;
```

Reply:
Great to know! I wasn't confident to have `validationEnabled` originally. happy to rename.